src/modules/game.js.js

```diff
@@ -0,0 +1,21 @@
+const createGame = async () => {
+  const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games';
+  const data = { name: "Kadare's Game" };
+
+  const request = new Request(url, {
+    method: 'POST',
+    headers: {
+      'content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+
+  const response = await fetch(request);
+
+  const responseData = await response.json();
+  // console.log(responseData);
```

Comment:
- [ ] It would be nice if you remove the commented piece of code to make your codebase clean to follow the JavaScript best practices as suggested by the previous reviewer :+1: